src/annotator/anchoring/text-range.ts

```diff
@@ -1,27 +1,23 @@
 /**
  * Return the combined length of text nodes contained in `node`.
- *
- * @param {Node} node
  */
-function nodeTextLength(node) {
+function nodeTextLength(node: Node): number {
   switch (node.nodeType) {
     case Node.ELEMENT_NODE:
     case Node.TEXT_NODE:
       // nb. `textContent` excludes text in comments and processing instructions
       // when called on a parent element, so we don't need to subtract that here.
 
-      return /** @type {string} */ (node.textContent).length;
+      return node.textContent?.length ?? 0;
```

Comment:
I presume this function should reliably return a `number`.

Reply:
Yes.